### PR TITLE
Implement Jan Jansen, Chaos Crafter

### PR DESCRIPTION
**Pull request by ZacharyDeganutti**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/jan_jansen_chaos_crafter.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Jan Jansen, Chaos Crafter
+ManaCost:R W B
+Types:Legendary Creature Gnome Artificer
+PT:3/3
+K:Haste
+SVar:TrigToken:DB$ Token | TokenScript$ c_1_1_a_construct
+A:AB$ Token | Cost$ T Sac<1/Creature.Artifact/artifact creature> | TokenAmount$ 2 | TokenScript$ c_a_treasure_sac | SpellDescription$ Create two Treasure tokens.
+A:AB$ Token | Cost$ T Sac<1/Artifact.nonCreature/noncreature artifact> | TokenAmount$ 2 | TokenScript$ c_1_1_a_construct | SpellDescription$ Create two 1/1 colorless Construct artifact creature tokens.
+Oracle:Haste\n{T}, Sacrifice an artifact creature: Create two Treasure tokens.\n{T}, Sacrifice a noncreature artifact: Create two 1/1 colorless Construct artifact creature tokens.
```
Comment (Agetian) [inline forge-gui/res/cardsfolder/upcoming/jan_jansen_chaos_crafter.txt:9]: SVar:AIPreference:SacCost$Artifact.nonCreature+Token+powerLE1+toughnessEQ1
DeckHas:Ability$Token|Sacrifice & Type$Treasure

File: forge-gui/res/cardsfolder/upcoming/jan_jansen_chaos_crafter.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Name:Jan Jansen, Chaos Crafter
+ManaCost:R W B
+Types:Legendary Creature Gnome Artificer
+PT:3/3
+K:Haste
+SVar:TrigToken:DB$ Token | TokenScript$ c_1_1_a_construct
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/jan_jansen_chaos_crafter.txt:6]: This TrigToken line seems unneeded?

Comment (ZacharyDeganutti) [reply forge-gui/res/cardsfolder/upcoming/jan_jansen_chaos_crafter.txt:6]: Think that was something from a script I used as a template that I forgot to clean up. Looks like I should be able to rip it out without any consequences. Going to double check now.

Comment (ZacharyDeganutti) [reply forge-gui/res/cardsfolder/upcoming/jan_jansen_chaos_crafter.txt]: Confirmed unnecessary and removed.

**Comment by tool4ever** [comment]
Looks good, thank you!

**Review by tool4ever — APPROVED** [review]

**Pull request merged**